packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -58,6 +64,19 @@ const Banners = () => {
     } else if (device?.features?.needs_backup) {
         banner = <NoBackup />;
         priority = 7;
+    } else if (device?.connected && device?.features?.safety_checks === 'PromptAlways') {
+        // PromptAlways could only be set via trezorctl. Warn user unconditionally.
+        banner = <SafetyChecksBanner />;
+        priority = 6;
```

Comment:
```suggestion
        priority = 50;
```

Hm, now it is not possible to insert banner between nobackup and updatebridge banner. So I multiplied existing priorities by 10.

FailedBackup = 90
NoBackup = 70
SafetyCheck = 50
UpdateBridge = 30
Update Firmware = 10

I will change it in message system today, so pull new version and change it to 50

Reply:
[Rebased and edited](https://github.com/trezor/trezor-suite/compare/fdab83a36049fdd373e8d9391c7b1dc946b33769..94386fd286a4f177e2508aa356dc4f3df0b5baf6`). Had to force push. Sorry for that.